internal/provider/provider.go

```diff
@@ -71,9 +75,30 @@ func (p *DagsterProvider) Configure(ctx context.Context, req provider.ConfigureR
 		return
 	}
 
+	var organization string
```

Comment:
It's a bit strange that you can configure partially with attributes and partially with envvars